tests/filecheck/dialects/arith/arith_invalid.mlir

```diff
@@ -19,9 +19,9 @@
 // -----
 
 %c = arith.constant 1 : si32
-// CHECK:  Unexpected attribute 1 : si32
+// CHECK: Expected attribute #builtin.signedness<signless> but got #builtin.signedness<signed>
 
 // -----
 
 %c = arith.constant 1 : ui32
-// CHECK:  Unexpected attribute 1 : ui32
```

Comment:
These seem to be the only test changes and in my opinion seem like a big improvement.